### ISSUE TITLE
Survey paper

### ISSUE THREAD

**Issue by bhack** [llvm/mlir-www#14]
Do you think It could be interesting to add this survey somewhere?https://arxiv.org/abs/2002.03794

**Comment by lattner** [comment]
It is great to see MLIR mentioned in this paper!  I think it would make sense to add this to something like llvm.org/pubs, which is for papers that mention LLVM related technologies.

**Comment by joker-eph** [comment]
Do you think we should merge MLIR-related paper with llvm.org/pubs instead of having a dedicated page on https://mlir.llvm.org/ (right now https://mlir.llvm.org/talks/ )? It'd be nice to have a way to tag publications that are about MLIR in the list of LLVM publications.

**Comment by lattner** [comment]
Yeah, I agree it would be nice to have a repository of MLIR related talks/papers separate from LLVM.  Either a tagging system or a different page would be great.  The llvm.org/pubs page is not exactly up to date either :)
